src/pages/quiz.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
Don't set global styles, this will propagate across all pages since we're dealing with a single page app.